diff --git a/node_modules/ecovacs-deebot/library/tools.js b/node_modules/ecovacs-deebot/library/tools.js
index d484ba1..1ad1f14 100644
--- a/node_modules/ecovacs-deebot/library/tools.js
+++ b/node_modules/ecovacs-deebot/library/tools.js
@@ -369,11 +369,13 @@ function envLogRaw(message) {
 }
 
 function logEvent(event, value) {
-    if (typeof value === 'object') {
-        console.log(chalk.bgGreen.white(' event ') + ' ' + chalk.green(event));
-        console.log(value);
-    } else {
-        console.log(chalk.bgGreen.white(' event ') + ' ' + chalk.green(event) + ' ' + value);
+    if ((process.env.NODE_ENV === 'development') || (process.env.NODE_ENV === 'dev')) {
+        if (typeof value === 'object') {
+            console.log(chalk.bgGreen.white(' event ') + ' ' + chalk.green(event));
+            console.log(value);
+        } else {
+            console.log(chalk.bgGreen.white(' event ') + ' ' + chalk.green(event) + ' ' + value);
+        }
     }
 }
 
